gflownet/evaluator/__init__.py

```diff
@@ -0,0 +1,340 @@
+"""
+An ``Evaluator`` is a class that is used to compute metrics and plots.
+It serves two complementary purposes:
+
+1. It is used to evaluate the performance of the agent during training and to log the
```

Comment:
```suggestion
1. Evaluate the performance of the agent during training and to log the
```

Reply:
Take all these suggestions in the text truly as suggestions (I am not implying the current version is wrong at all)